packages/webapp/src/containers/Animals/AddAnimals/AddAnimalDetails/usePopulateDetails.ts

```diff
@@ -86,7 +92,11 @@ export const usePopulateDetails = (
 
       return updatedData.map((entity, index) => {
         const origData = trimmedOrigGroup[index] || {};
-        return { ...origData, ...entity };
+        return {
+          ...origData,
+          ...entity,
+          [DetailsFields.ORGANIC_STATUS]: origData[DetailsFields.ORGANIC_STATUS],
```

Comment:
Its too bad this goes here -- it was so clean before haah

Reply:
It doesn't have to go here! We can use the `useEffect()` method (https://github.com/LiteFarmOrg/LiteFarm/pull/3398#discussion_r1735330899) instead!!

Let's see if that fixes the bug too because that would be easiest (yes, I do see it)!